packages/grafana-data/package.json

```diff
@@ -56,25 +59,18 @@
     "@types/node": "16.11.6",
     "@types/papaparse": "5.2.0",
     "@types/react": "17.0.30",
-    "@types/rollup-plugin-visualizer": "4.2.1",
+    "@types/react-dom": "17.0.10",
     "@types/sinon": "^10.0.4",
     "@types/testing-library__jest-dom": "5.9.5",
     "@types/testing-library__react-hooks": "^3.2.0",
     "@types/tinycolor2": "1.4.2",
-    "pretty-format": "25.1.0",
-    "react": "17.0.1",
-    "react-dom": "17.0.1",
     "react-test-renderer": "17.0.1",
-    "regenerator-runtime": "0.13.3",
```

Comment:
I think this package is needed, otherwise I get this error after running `yarn start`
```
Error: Your application tried to access regenerator-runtime, but it isn't declared in your dependencies; this makes the require call ambiguous and unsound.

Required package: regenerator-runtime
Required by: /Users/alexkhomenko/dev/go/src/github.com/grafana/grafana/packages/grafana-data/src/vector/

```

Reply:
great catch @Clarity-89, thanks! 🙌